octoprint_discordremote/command.py

```diff
@@ -443,25 +442,28 @@ def unzip(self, params):
                         combined.write(temp.read())
                     self.plugin.get_file_manager().remove_file('local', f.rpartition('/')[2])
 
-
             unzippable = upload_file_path
 
         else:
-            return None, error_embed(author=self.plugin.get_printer_name(), title="Not a valid Zip file.", description='try "%sunzip [filename].zip or %sunzip [filename].zip.001 for multi-volume files."' % (self.plugin.get_settings().get(
+            return None, error_embed(author=self.plugin.get_printer_name(),
+                                     title="Not a valid Zip file.",
+                                     description='try "%sunzip [filename].zip or %sunzip [filename].zip.001 for multi-volume files."' % (self.plugin.get_settings().get(
                                          ["prefix"]), self.plugin.get_settings().get(
                                          ["prefix"])))
 
         if unzippable == None:
             return None, error_embed(author=self.plugin.get_printer_name(), title="File %s not found." % file_name)
 
-
         try:
             with zipfile.ZipFile(unzippable) as zip:
 
                 fileOK = zip.testzip()
 
                 if fileOK is not None:
-                    return None, error_embed(author=self.plugin.get_printer_name(), title="Bad zip file.", description='In case of multi-volume files, one could be missing.')
+                    self.plugin.get_file_manager().remove_file('local', unzippable)
```

Comment:
this error case is the same as the exception, perhaps raise an exception to avoid the duplication?

Reply:
Good point, I'll do that.